googlemaps/geocoding.py

```diff
@@ -90,7 +90,10 @@ def reverse_geocode(client, latlng, result_type=None, location_type=None,
     :rtype: list of reverse geocoding results.
     """
 
-    params = {"latlng": convert.latlng(latlng)}
+    if (convert.is_string(latlng) and (',' not in latlng)):
```

Comment:
Almost there - most of those parentheses aren't necessary, so just: 

`if convert.is_string(latlng) and ',' not in latlng:`

Could probably also use a comment mentioning why we check for a comma